openupgrade_scripts/scripts/sale_coupon/15.0.1.0/post-migration.py

```diff
@@ -6,10 +6,3 @@
 @openupgrade.migrate()
 def migrate(env, version):
     openupgrade.load_data(env.cr, "sale_coupon", "15.0.1.0/noupdate_changes.xml")
```

Comment:
Comment this line. The file is not needed because only modifies a template, but the template is not noupdate anymore.